tsdb/shard.go

```diff
@@ -465,26 +471,26 @@ func (s *Shard) validateSeriesAndFields(points []models.Point) ([]*FieldCreate,
 	for _, p := range points {
 		// verify the tags and fields
 		tags := p.Tags()
-		if v := tags.Get([]byte("time")); v != nil {
+		if v := tags.Get(staticTimeByteSlice); v != nil {
 			s.logger.Printf("dropping tag 'time' from '%s'\n", p.PrecisionString(""))
-			tags.Delete([]byte("time"))
+			tags.Delete(staticTimeByteSlice)
 			p.SetTags(tags)
 		}
 
 		fields := p.Fields()
-		if _, ok := fields["time"]; ok {
+		if _, ok := fields[staticTimeString]; ok {
```

Comment:
`staticTimeString` is not needed, since Go will only add the string literal once in the binary, and each instance will refer to the same one. A new string will not be allocated multiple times.